bokeh/embed.py

```diff
@@ -386,7 +386,8 @@ def file_html(models,
               resources,
               title=None,
               template=FILE,
-              template_variables={}):
+              template_variables={},
+              theme=None):
```

Comment:
It should be ``theme=FromCurdoc`` so that the behaviour matches bokeh.embed.components.